pos_default_payment_method/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Not blocking, but recommended to use readme fragmentes (https://github.com/OCA/maintainer-tools/tree/master/template/module/readme) as it makes the readme more maintainable through versions.